resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnSchedulerBackend.scala

```diff
@@ -161,6 +172,35 @@ private[spark] abstract class YarnSchedulerBackend(
     totalRegisteredExecutors.get() >= totalExpectedExecutors * minRegisteredRatio
   }
 
+  override def getShufflePushMergerLocations(
```

Comment:
How about moving this to `CoarseGrainedSchedulerBackend`? The implementation doesn't seem to be specific to YARN.

Reply:
@Ngone51 Currently push based shuffle is only supported for Yarn and not for other cluster managers. Would it make sense to move this to `CoarseGrainedSchedulerBackend` once we have it supported for other cluster managers?

Reply:
Ok, sounds good.